.gitpod.yml

```
@@ -12,7 +12,7 @@ ports:
   - port: 9339 # Node.js debug port
     onOpen: ignore
 tasks:
-  - init: yarn --network-timeout 100000
+  - init: yarn --network-timeout 100000 && yarn build:examples
```

I was wondering why my Theia installation in gitpod didn't run any extensions. I think we should download extensions here as well:

```suggestion
  - init: yarn --network-timeout 100000 && yarn build:examples && yarn download:plugins
```

---

Ah good catch!